go.mod

```diff
@@ -3,59 +3,41 @@ module github.com/nsheridan/cashier
 go 1.16
```

Comment:
You need to rebase off master, this is a way old mod file and it's the cause of the broken test